plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -889,10 +901,12 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 
 	/* setup firmware parameters */
 	switch (self->chip.model) {
-	case ISP_MODEL_HUB_GL3523:
+	case ISP_MODEL_HUB_GL3523: {
+		gint var = 0;
```

Comment:
I tend to use `tmp` for temporary variables; it as more sense that just `var`.

However, `bonding` sounds better.

Reply:
Ok, that's do it.